pkgs/build-support/trivial-builders.nix

```diff
@@ -258,15 +258,17 @@ rec {
    * Example:
    *
    * # Symlinks hello path in store to current $out/hello
-   * linkFarm "hello" entries = [ { name = "hello"; path = pkgs.hello; } ];
+   * linkFarm "hello" [ { name = "hello"; path = pkgs.hello; } ];
    *
    */
   linkFarm = name: entries: runCommand name { preferLocalBuild = true; }
-    ("mkdir -p $out; cd $out; \n" +
-      (lib.concatMapStrings (x: ''
-        mkdir -p "$(dirname '${x.name}')"
-        ln -s '${x.path}' '${x.name}'
-      '') entries));
+    ''mkdir -p $out
```

Comment:
Maybe also double-quote `$out`?

Reply:
```$out``` should be safe, its character set is limited.

Reply:
Indeed it should, until someone manages to run a store in a space-containing path (right now this would probably not work regardless of this, I agree)

Reply:
Yes, I was thinking in placing nix store under ```C:\Program Files\``` :)